### PR TITLE
Replace initial with an actual height

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description
This PR replaces the `initial` value of `$navbar-height` with an actual value because it's used in `calc` expressions in the compiled CSS. This faulty CSS is then imported in Vue Vocabulary where it breaks the build in a stunning demonstration of the butterfly effect.

## Technical details
The PostCSS config in Vue Vocabulary prevents faulty CSS, such as this, from passing through the build process.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/styles/header.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-$navbar-height: initial;
+$navbar-height: 5.0625rem;
```
Comment (brenoferreira) [inline src/styles/header.scss]: The small header in the Design Library has a height of 82px or 5.125rem

**Review by brenoferreira — CHANGES_REQUESTED** [review]

**Comment by brenoferreira** [comment]
Screen Shot 2020-04-09 at 10 45 46

_there's also some issue causing the logo to be  misaligned with the menu on the right_

File: src/styles/header.scss (hunk 1 above)
Comment (dhruvkb) [reply src/styles/header.scss]: Removed it altogether as setting it to a fixed value creates more problems than it solves.

**Comment by dhruvkb** [comment]
@brenoferreira let me know if this is better?

**Review by hugosolar — APPROVED** [review]

**Review by brenoferreira — APPROVED** [review]

**Pull request merged**